jekyll/_cci2/executor-types.md

```diff
@@ -332,8 +331,7 @@ version: 2
 
 jobs:
   build: # name of your job
-    machine:
-      image: windows-default # Windows machine image
```

Comment:
We'll need the image back on this one as it's the Windows example (sorry again!)